thirdparty/spdlog/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+include(FetchContent)
+
+FetchContent_Declare(
+    spdlog
+    GIT_REPOSITORY https://github.com/gabime/spdlog.git
+    GIT_TAG        v1.6.0
```

Can you verify whether or not it works with `GIT_SHALLOW true`? This avoids fetching the entire git history for a dependency which in some cases is the majority of the download cost.

```suggestion
    GIT_TAG        v1.6.0
    GIT_SHALLOW true
```